Sources/LiveViewNative/Coordinators/LiveViewCoordinator.swift

```diff
@@ -298,6 +300,28 @@ public class LiveViewCoordinator<R: RootRegistry>: ObservableObject {
         }
     }
     
+    private func getPlatformMetadata() throws -> String {
+        let encoder = JSONEncoder()
+        let metadata: [String : String] = [
+            "platform_variant": getPlatformVariant()
+        ]
+        let encodedMetadata = try encoder.encode(metadata)
+
+        return String(data: encodedMetadata, encoding: .utf8)!
+    }
+    
+    private func getPlatformVariant() -> String {
+        #if os(macOS)
+        return "macOS"
+        #elseif os(tvOS)
+        return "tvOS"
+        #elseif os(watchOS)
+        return "watchOS"
+        #else
+        return UIDevice.current.localizedModel == "iPhone" ? "iOS" : "iPadOS"
```

Comment:
@bcardarella That _is_ something we could pass along to the server, although there might be more to that since what happens when a user rotates the orientation of their device? Do we attempt to listen for that and send an updated `platformMetadata` somehow when it happens?

Reply:
In general its better to design the UI based on the [`UserInterfaceSizeClass`](https://developer.apple.com/documentation/swiftui/userinterfacesizeclass), rather than a particular interface idiom. For example, when you open an iPad app in split view it will have a `horizontalSizeClass` of `compact`, which is the same as a normal iOS app. In that case, the UI would adjust to behave like on iOS.

Reply:
That makes sense @carson-katri. Do you think my comment above about updating the server's platform metadata on resizes would still apply? I'm thinking about iPadOS which allows you to have two apps open side-by-side and resize them or like newer iPads that have Stage Manager.

Reply:
@carson-katri Agreed. Let's address that as a separate issue for now as it requires more changes than I'd like to introduce with this PR.